src/System.Net.Http/src/project.json

```diff
@@ -12,6 +12,7 @@
     "System.Runtime.Handles": "4.0.0",
     "System.Runtime.InteropServices": "4.0.20",
     "System.Security.Cryptography.X509Certificates": "4.0.0-*",
+    "System.Security.Cryptography.OpenSsl": "4.0.0-*",
```

Comment:
Everywhere else the `*` versions require beta, e.g. `4.0.0-beta-*`, this should probably follow that pattern (also for the X509Certificates reference).